src/adapters.ts

```diff
@@ -2283,6 +2283,63 @@ export function AdapterL2<TBase extends Constructor<TxSender>>(Base: TBase) {
       return customData;
     }
 
+    /**
+     * Returns the amount of approved tokens for a specific L2 bridge.
+     *
+     * @param token The address of the token.
+     * @param [bridgeAddress] The address of the bridge contract to be used.
+     * Defaults to the default L2 shared bridge.
+     * @param [blockTag] The block in which an allowance should be checked.
+     * Defaults to 'committed', i.e., the latest processed block.
+     */
+    async getAllowanceL2(
```

Comment:
I don't really like this and the `approveERC20L2` methods. 
1. They are bridge-specific, but they don't tell that through the signature. If I call `wallet.getAllowanceL2(token)`, I don't expect it to get allowance associated with the default bridge.
2. They don't seem useful enough for a general user -- outside of _implementing_ bridge functionality (responsibility of the library author, not user), I don't expect users to ever call them.

I think it'll be better to remove default bridge assumption from them (e.g. you always have to know the bridge address if you want to call them) and move it to bridges.

If you will have an abstract base class for bridges, it can be a provided method there. If not, you can simply make them private methods on the USDC bridge.

Reply:
Agreed, moved them to the abstract `Bridge` class and removed default bridge assumption.